app/hooks/useQuestionStateInUrl.ts

```diff
@@ -1,22 +1,25 @@
 import {useState, useRef, useEffect, useMemo, useCallback} from 'react'
 import type {MouseEvent} from 'react'
 import {useSearchParams, useTransition} from '@remix-run/react'
-import {Question, QuestionState} from '~/server-utils/stampy'
+import {Question, QuestionState, RelatedQuestions} from '~/server-utils/stampy'
 import {fetchOnSiteAnswers} from '~/routes/questions/allOnSite'
 
-const getStateEntries = (state: string): [string, QuestionState][] =>
+type PageId = Question['pageid']
+
+const getStateEntries = (state: string): [PageId, QuestionState][] =>
   Array.from(state.matchAll(/([^-_r]+)([-_r]*)/g) ?? []).map((groups) => [
-    groups[1],
-    (groups[2] || '_') as QuestionState,
+    groups[1], // question id
+    (groups[2] as QuestionState) || QuestionState.OPEN,
```

Comment:
I was finding it hard to remember what char meant what, so I went and made an enum for it